APMrover2/Parameters.cpp

```diff
@@ -603,6 +603,24 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Path: ../libraries/AC_Avoidance/AP_OAPathPlanner.cpp
     AP_SUBGROUPINFO(oa, "OA_", 45, ParametersG2, AP_OAPathPlanner),
 
+    // @Param: MAX_SPEED
+    // @DisplayName: Maximum vehicle speed
+    // @Description: Maximum speed vehicle can obtain at full throttle. If 0, it will be estimated based on CRUISE_SPEED and CRUISE_THROTTLE.
+    // @Units: m/s
+    // @Range: 0 100
+    // @Increment: 0.1
+    // @User: Advanced
+    AP_GROUPINFO("MAX_SPEED", 46, ParametersG2, max_speed, 0.0f),
+
+    // @Param: LOIT_SPEED_GAIN
+    // @DisplayName: Loiter speed gain
+    // @Description: Determines how agressively LOITER tries to correct for drift from loiter point. Higher is faster but default should be acceptable.
+    // @Units: m/s
```

Comment:
We should remove the @Units 'cuz it's probably unit-less.  Maybe the Range should be reduced as well to be 0 to 5.
Increment could perhaps be 0.01.

Reply:
change made